client/signup/config/flows-pure.js

```diff
@@ -340,17 +340,6 @@ export function generateFlows( {
 			lastModified: '2020-08-11',
 			showRecaptcha: true,
 		},
-		// IMPORTANT: steps should match the onboarding flow through the `site-type` step to prevent issues
-		// when switching from the onboarding flow.
-		{
-			name: 'import-onboarding',
-			steps: [ 'user', 'site-type', 'import-url', 'import-preview', 'domains', 'plans-import' ],
-			destination: getImportDestination,
```

Comment:
Did we remove references to `getImportDestination` ..? Is that used anywhere else?

Reply:
It's used on another `import` route. This is all legacy code related to the old "blue" pages that can be removed once we remove everything.